pyproject.toml

```
@@ -168,21 +168,56 @@ networkx = ['tests/*.py']
 "networkx.utils" = ['tests/*.py']
 
 [tool.ruff.lint]
-select = [
-    'I',
-    'UP',
-    'C4',
-    'E713',
-    'PIE',
-    'PGH003',
-    'PLR0402',
-    'SIM101',
-    'SIM109',
-    'SIM110',
-    'SIM111',
-    'SIM118',
-    'SIM2',
+extend-select = [
+#  "B",        # flake8-bugbear
+  "I",        # isort
+#  "ARG",      # flake8-unused-arguments
+  "C4",       # flake8-comprehensions
+#  "D",        # pydocstyle (see tool.ruff.lint.pydocstyle below)
+  "EM",       # flake8-errmsg
+  "ICN",      # flake8-import-conventions
+#  "G",        # flake8-logging-format
+  "PGH",      # pygrep-hooks
+  "PIE",      # flake8-pie
+#  "PL",       # pylint
+  "PLR0402",
+#  "PT",       # flake8-pytest-style
+#  "PTH",      # flake8-use-pathlib
+#  "RET",      # flake8-return
+#  "RUF",      # Ruff-specific
+#  "SIM",      # flake8-simplify
+   "SIM101",
+   "SIM109",
+   "SIM110",
+   "SIM111",
+   "SIM118",
+   "SIM2",
```

These are indented too much, and `SIM111` has been remapped to `SIM110`.

```suggestion
  "SIM101",
  "SIM109",
  "SIM110",
  "SIM118",
  "SIM2",
```